modules/common-fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/Run.scala

```diff
@@ -147,8 +149,14 @@ object Run {
   ): F[ExitCode] =
     environment.use { env =>
       val log = Logger[F].info("Running enrichment stream")
-      val enrich = Enrich.run[F, A](env)
-      val updates = Assets.run[F, A](env.blocker, env.semaphore, env.assetsUpdatePeriod, env.assetsState, env.enrichments)
+      val enrich = {
+        implicit val remoteAdapterHttpClient: Http4sClient[F] = env.remoteAdapterHttpClient
+        Enrich.run[F, A](env)
+      }
+      val updates = {
+        implicit val httpClient: Http4sClient[F] = env.httpClient
+        Assets.run[F, A](env.blocker, env.semaphore, env.assetsUpdatePeriod, env.assetsState, env.enrichments)
+      }
```

Comment:
```suggestion
implicit val httpClient: Http4sClient[F] = env.httpClient
val enrich = Enrich.run[F, A](env)
val updates = Assets.run[F, A](env.blocker, env.semaphore, env.assetsUpdatePeriod, env.assetsState
```
Don't only the remote adapters need `remoteAdaptersHttpClient` ?